dockerfiles/che/Dockerfile

```diff
@@ -5,13 +5,22 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-FROM openjdk:8u242-jre-slim
+
+# Variables in `COPY --from=` is not supported see https://github.com/moby/moby/issues/34482
+# this is workaround to handle that.
+ARG CHE_DASHBOARD_VERSION=next
+ARG CHE_WORKSPACE_LOADER_VERSION=next
+
+FROM quay.io/eclipse/che-dashboard:${CHE_DASHBOARD_VERSION} as che_dashboard_base
+FROM quay.io/eclipse/che-workspace-loader:${CHE_WORKSPACE_LOADER_VERSION} as che_workspace_loader_base
+
+FROM openjdk:8u252-jre-slim
 ENV LANG=C.UTF-8
-RUN echo "%root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && rm -rf /tmp/*
```

Comment:
you're not cleaning out /tmp ?

Reply:
there is nothing there but I can revert this change if you prefer.